.github/workflows/sam_release_code.yml

```diff
@@ -147,6 +147,12 @@ jobs:
           echo "Specification location: $SPEC_PATH"
           echo "SPEC_PATH=${SPEC_PATH}" >> "$GITHUB_ENV"
       
+      - name: Set ECR Repository
+        run: echo "ECR_REPOSITORY=fhir-facade-repo" >> "$GITHUB_ENV"
+
+      - name: Set Image Tag
+        run: echo "IMAGE_TAG=${{ inputs.VERSION_NUMBER }}" >> "$GITHUB_ENV"
```

Comment:
Can we just inline these below? I think this just makes it harder to figure out what values are being used when reading the code.

Reply:
The values for `fhir-facade-repo` and `validator-repo` are used directly in the `ECSTaskDefinition` resource.